nominal/core/_clientsbunch.py

```diff
@@ -27,9 +27,54 @@
     timeseries_logicalseries,
     upload_api,
 )
+from requests.adapters import (
+    Response,
+)
 from typing_extensions import Self
 
 
+class ProtoWriteService(storage_writer_api.NominalChannelWriterService):
+    def write_nominal_batches(self, auth_header: str, data_source_rid: str, request: Any) -> None:
+        _headers: Dict[str, Any] = {
+            "Accept": "application/json",
+            "Content-Type": "application/x-protobuf",
+            "Authorization": auth_header,
+        }
```

Comment:
I understand this was typed like this in the generated conjure, but we can/should be more specific here--let's use the correct protobuf type, not use Any, etc. Also I don't think these annotations on the types inside the function are necessary at all.